.eslintignore

```diff
@@ -1,2 +1,3 @@
 node_modules
 coverage
+spec/react_on_rails/dummy-for-generators
```

Comment:
@justin808 it's just a tmp directory that the tests clean out every time they run their code. Some of the files in there are just simulations that don't have true ruby code in them and would give off all sorts of false flags in the linters. In addition, whatever is in there is *only* the setup for whatever test happened to be run last. I think a much better idea is to implement linters in our generator-results project, right?